applications/kserve/models-web-app/overlays/kubeflow/kustomization.yaml

```diff
@@ -16,6 +16,7 @@ configMapGenerator:
 - behavior: replace
   literals:
   - USERID_HEADER=kubeflow-userid
+  - USERID_PREFIX=""
```

Comment:
That seems to be missing upstream. https://github.com/kserve/models-web-app/blob/a3e7525ffcae2f5d9e0babcfd59f7d7ed9d665a3/config/overlays/kubeflow/kustomization.yaml#L18

Reply:
See https://github.com/kserve/models-web-app/pull/137